include/picongpu/fields/incidentField/Functors.hpp

```diff
@@ -336,7 +376,7 @@ namespace picongpu
                         auto const& subGrid = Environment<simDim>::get().SubGrid();
                         auto const globalDomainCells = subGrid.getGlobalDomain().size;
                         auto const direction = getDirection();
-                        auto const focus = getFocus();
+                        auto const focus = projectFocus();
```

Comment:
For future refactoring: This shadows the struct's member variable. Yet it is set to the same value...